generator/tree_test.go

```diff
@@ -262,6 +262,55 @@ func TestGenerateConfigModule(t *testing.T) {
 				},
 			},
 		},
+		// Tables with accessible & inaccessible.
+		{
+			node: &Node{Oid: "1", Label: "root",
+				Children: []*Node{
+					{Oid: "1.1", Label: "table",
+						Children: []*Node{
+							{Oid: "1.1.1", Label: "tableEntry",
+								Children: []*Node{
+									{Oid: "1.1.1.1", Access: "ACCESS_NOACCESS", Label: "tableNoAccessIndex", Type: "INTEGER"},
+									{Oid: "1.1.1.2", Access: "ACCESS_CREATE", Label: "tableCreateDescr", Type: "OCTETSTR"},
```

Comment:
I'd make these all of type integer to keep focused on what we're actually testing here.